manifests/kustomize/env/cert-manager/base/webhook/pipelineversion-cert-issuer.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: pipelineversion-webhook-selfsigned-issuer
```

Comment:
Could you use more generic names such as `kfp-api-webhook-selfsigned-issuer` or similar since we'll likely add more webhooks in the future and we can reuse the certificate.